doc/source/guide.rst

```diff
@@ -368,6 +368,78 @@ or a relative path to the real path::
     (see our :doc:`contributing guide <contributing>`).
 
 
+Configuration Files
+-------------------
+
+.. warning::
+    Config files are an experimental feature
+    and may be subject to change without prior notice.
+
+Defaults for the command line options can be set in a configuration file.
+Example::
+
+    filter = src/
+    html-details = yes  # info about each source file
+    output = build/coverage.html
+
+How the configuration file is found:
+If a :option:`--config` option is provided, that file is used.
+Otherwise, a ``gcovr.cfg`` file in the :option:`--root` directory is used.
+If the above steps name a non-existent config file,
+this is ignored silently.
+
+The configuration file uses an INI-style syntax.
```

Comment:
The only tweak I'd make would be to avoid mentioning INI syntax altogether and just call it it a key value store that stacks duplicate keys when appropriate.

Reply:
Ok, I rephrased that part. Thanks for reviewing this!